src/mgr.sol

```diff
@@ -192,54 +188,49 @@ contract TinlakeManager {
 
     // moves the rwaToken into the vault
     // requires that mgr contract holds the rwaToken
-    function lock(uint wad) public operatorOnly {
+    function lock(uint256 wad) public auth {
         require(vat.live() == 1, "TinlakeManager/mkr-in-ES");
+        rwaToken.approve(address(urn), uint256(wad));
         urn.lock(wad);
-        live = true;
```

Comment:
any subsequent call will fail without calling lock first but moving it up into the modifier makes more sense as it doesn't create some weird pre-launch state that is now also tracked via `live`